src/iidm/NodeTerminalViews.cpp

```diff
@@ -44,6 +44,10 @@ stdcxx::Reference<Bus> BusBreakerViewImpl::getConnectableBus() {
     return voltageLevel.getCalculatedBusBreakerTopology().getConnectableBus(m_terminal.getNode());
 }
 
+void BusBreakerViewImpl::moveConnectable(const std::string& busId, bool connected) {
+    m_terminal.getConnectable().get().move(m_terminal, m_terminal.getConnectionInfo(), busId, connected);
```

Comment:
`throw AssertionError("Not implemented");` instead ?

Reply:
I don't understand why you suggest to throw an exception?